buildconfig/CMake/PythonPackageTargetFunctions.cmake

```diff
@@ -59,21 +63,35 @@ CustomInstallLib = patch_setuptools_command('install_lib')
     DEPENDS ${_outputs}
   )
 
+
   if ( ${PACKAGE_WORKBENCH} )
     # setuptools by default wants to build into a directory called 'build' relative the to the working directory. We have overridden
     # commands in setup.py.in to force the build directory to take place out of source. The install directory is specified here and then
     # --install-scripts=bin --install-lib=lib removes any of the platform/distribution specific install directories so we can have a flat
     # structure
     install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${_setup_py} install -O1 --single-version-externally-managed --root=${_setup_py_build_root}/install --install-scripts=bin --install-lib=lib WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})")
-    # register the "installed" components with cmake so it will carry them over
-    install(DIRECTORY ${_setup_py_build_root}/install/lib
-            DESTINATION .
+
+    # Specify the installation directory based on OS
+    if ( WIN32 )
```

Comment:
I don't like having this additional check here, but putting the logic in the `if (WIN32)` above did not work, as the function is not always called with the `EXECUTABLE` parameter, but we always need to set the correct installation directory. 

I chose to set the install destination in a variable, to avoid duplicating the more complicated `install` command below. I have not tested if `set ( _package_install_destination .)` actually works as intended on Linux

Reply:
I can test that out.